.github/workflows/test-job.yaml

```diff
@@ -379,3 +379,32 @@ jobs:
         with:
           ray_version: 2.3.0
 
+  python-client-test:
+    runs-on: ubuntu-latest
+    name: Python Client Test
+    steps:
+      - name: Set up Docker
+        uses: docker-practice/actions-setup-docker@master
+
+      - name: Install Kind
```

Comment:
Would you mind sharing more details about why we need to create a k8s cluster here? Thanks!